presto-main/src/main/java/com/facebook/presto/server/security/LdapFilter.java

```diff
@@ -119,7 +122,14 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
                 environment.put(SECURITY_CREDENTIALS, password);
                 context = authenticate(environment);
 
-                //TODO: Use the context to do the group membership search
+                if (groupDistinguishedName.isPresent()) {
+                    if (!ldapBinder.checkForGroupMembership(user, groupDistinguishedName.get(), context)) {
+                        String errorMessage = format("User %s not a member of the group %s.", user, groupDistinguishedName.get());
+                        LOG.debug("Authorization failed for user. " + errorMessage);
```

Comment:
Remove this logging -- there is no additional information, such as an exception from the LDAP server